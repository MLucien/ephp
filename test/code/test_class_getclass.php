<?php

class A { }

$a = new A;
$classname = "A";

print A::class . "\n";
print "get_class = " . get_class($a) . "\n";

var_dump(get_class(array()));
