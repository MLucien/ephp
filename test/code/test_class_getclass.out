A
get_class = A

Warning: get_class() expects parameter 1 to be object, array given in {{CWD}}/test/code/test_class_getclass.php on line 11
bool(false)
